test/functional/ws/transport.transactions.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+var node = require('../../node');
+var shared = require('../http/post/shared');
+var ws = require('../../common/wsCommunication');
+
+function postTransaction (transaction, done) {
+	ws.call('postTransactions', {
+		transaction: transaction
+	}, done, true);
+}
+
+describe('Posting transaction (type 0)', function () {
+
+	var transaction;
+	var goodTransactions = [];
+	var badTransactions = [];
+	var account = node.randomAccount();
+
+	beforeEach(function () {
+		transaction = node.randomTx();
+	});
+
+	describe('transaction processing', function () {
+
+		it('when sender has no funds should fail', function (done) {
+			var transaction = node.lisk.transaction.createTransaction('1L', 1, account.password);
+
+			postTransaction(transaction, function (err, res) {
+				node.expect(res).to.have.property('success').to.be.not.ok;
+				node.expect(res).to.have.property('message').to.equal('Account does not have enough LSK: ' + account.address + ' balance: 0');
```

Comment:
this has `+ var +` but other entries are `+var`.

We should make sure that we follow the same spacing standards everywhere. In this case the standard is right here but wrong elsewhere.

Here are some examples from the code base on the format.

https://github.com/LiskHQ/lisk/blob/development/logic/peers.js#L115
https://github.com/LiskHQ/lisk/blob/development/logic/transactionPool.js#L425
https://github.com/LiskHQ/lisk/blob/development/logic/transactionPool.js#L458